lib/charms/opensearch/v0/opensearch_relation_peer_cluster.py

```diff
@@ -1445,3 +1484,56 @@ def _clear_errors(self, *error_labels: str):
             error = self.charm.peers_data.get(Scope.APP, error_label, "")
             self.charm.status.clear(error, app=True)
             self.charm.peers_data.delete(Scope.APP, error_label)
+
+    def set_first_data_node(self, first_data_node: str) -> None:
```

Comment:
maybe better
```suggestion
    def set_first_data_node(self, first_data_node: str | None) -> None:
```

and if None, use `delete_from_rel` instead of `put_in_rel`